Sensory/src/main/AndroidManifest.xml

```diff
@@ -1,4 +1,19 @@
 <?xml version="1.0" encoding="utf-8" ?>
+<!--
+ Copyright 2020 Google LLC
```

Comment:
Should the year be `2024` rather than `2020`?

Reply:
thanks done.